operator-pipeline-images/operatorcert/entrypoints/update_preflight_versions.py

```diff
@@ -0,0 +1,182 @@
+"""
```

Comment:
disable_preflight_versions or invalidate_preflight_versions would probably be a more precise name for the script? Update is a bit too general.